model/src/test/kotlin/config/OrtConfigurationTest.kt

```diff
@@ -310,20 +302,15 @@ class OrtConfigurationTest : WordSpec({
         "support references to environment variables" {
             val configFile = createTestConfig(
                 """
-                ort {
-                  scanner {
-                    storages {
-                      postgresStorage {
-                        connection {
-                          url = "postgresql://your-postgresql-server:5444/your-database"
-                          schema = "public"
-                          username = ${'$'}{POSTGRES_USERNAME}
-                          password = ${'$'}{POSTGRES_PASSWORD}
-                        }
-                      }
-                    }
-                  }
-                }
+                ort:
+                  scanner:
+                    storages:
+                      postgresStorage:
+                        connection:
+                          url: "postgresql://your-postgresql-server:5444/your-database"
+                          schema: public
+                          username: ${'$'}{POSTGRES_USERNAME}
+                          password: ${'$'}{POSTGRES_PASSWORD}
```

Comment:
Just checking: Is this syntax really still correct with YAML?

Reply:
It's required for Kotlin to escape `$` in a string literal.